DDCore/src/segmentations/HexGrid.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ * HexGrid.cpp
+ *
+ *  Created on: August 9, 2023
+ *      Author: Sebouh J. Paul, UC Riverside
+ */
+#include "DD4hep/Factories.h"
+#include "HexGrid.h"
+
+namespace dd4hep {
+  namespace DDSegmentation {
+
+    /// Default constructor used by derived classes passing the encoding string
+    HexGrid::HexGrid(const std::string& cellEncoding) :
+      Segmentation(cellEncoding) {
+      _type = "HexGridXY";
+	_description = "Hexagonal segmentation in the local XY-plane";
```

Comment:
Indentation is different here to previous line

Reply:
fixed with commit [84373a2](https://github.com/sebouh137/DD4hep/commit/84373a217539ac024beafacec1501327381224ab)